### PR TITLE
 Introduce option: devServer

### PR DESCRIPTION
**Pull request by dtslvr**

### CODE REVIEW

File: packages/cubejs-server-core/core/index.js [hunk 1]
```diff
@@ -81,6 +81,7 @@ class CubejsServerCore {
       driverFactory: () => CubejsServerCore.createDriver(options.dbType),
       apiSecret: process.env.CUBEJS_API_SECRET,
       dbType: process.env.CUBEJS_DB_TYPE,
+      devServer: true,
```
Comment (paveltiunov) [inline packages/cubejs-server-core/core/index.js]: To ensure backward compatibility should be `devServer: process.env.NODE_ENV !== 'production'`

File: packages/cubejs-server-core/core/index.js [hunk 2]
```diff
@@ -162,7 +163,9 @@ class CubejsServerCore {
       this.logger
     );
     apiGateway.initApp(app);
-    this.initDevEnv(app);
+    if (this.options.devServer) {
```
Comment (paveltiunov) [inline packages/cubejs-server-core/core/index.js]: There're already these devServer checks in place. I believe we should just replace all `process.env.NODE_ENV !== 'production'` occurrences with `this.options.devServer`.

Comment (dtslvr) [reply packages/cubejs-server-core/core/index.js]: I agree, please see https://github.com/statsbotco/cube.js/pull/29/files#diff-49fd186d45485663b0e641a0214512e2R102 (Line 102)

File: packages/cubejs-server-core/core/index.js (hunk 1 above)
Comment (dtslvr) [reply packages/cubejs-server-core/core/index.js]: Done.

**Comment by dtslvr** [comment]
Please check my improvements after the code review.

File: packages/cubejs-server-core/core/index.js [hunk 3]
```diff
@@ -162,45 +163,45 @@ class CubejsServerCore {
...
+      this.initDevEnv(app);
+    }
   }
 
   initDevEnv(app) {
-    if (process.env.NODE_ENV !== 'production') {
```
Comment (dtslvr) [inline packages/cubejs-server-core/core/index.js]: I have removed this check, I prefer it at the function call.

**Comment by paveltiunov** [comment]
Looks great!

**Pull request merged**